### DISCUSSION TITLE
I feel a bit guilty of copying the top solutions in open targets....

### DISCUSSION THREAD

**Discussion by joewinbpaul** [cssbattle/forum#336] [General]
Every time u open an open target u immediately copy the top solution but won't even bother to do it yourself (unless u r a pro and want to break the record). <p>  I feel like something should be done about this <p> Maybe something like u cant access the top solutions in open targets unless u get a 100% match yourself first (just an idea) <p> By doing this you can know whether you deserve the points or not

**Comment by joewinbpaul** [comment]
Do you happen to have any better ideas???

**Reply by rafehimself** [reply]
many ideas and problems have been presented over the years,
the problem is that there is little if any responses to the feedback 🙄

**Comment by Beowolve** [comment]
I really like that Idea. Maybe a simple timer would work too. For example show the submissions only after a specific time has past.
Let's say 3 days, that would also prevent everyone from  copy pasting new records seconds after the first submission

**Reply by joewinbpaul** [reply]
> I really like that Idea. Maybe a simple timer would work too. For example show the submissions only after a specific time has past. Let's say 3 days, that would also prevent everyone from copy pasting new records seconds after the first submission

Great idea!

**Reply by joewinbpaul** [reply]
I just hope the maintainer sees this and does something.....

**Comment by joewinbpaul** [comment]
Anyone else got better ideas??

**Reply by Esger** [reply]
Yes, the open targets shouldn’t be opened before the end of the battle.

**Reply by joewinbpaul** [reply]
> Yes, the open targets shouldn’t be opened before the end of the battle.

..... nice!

**Reply by rafehimself** [reply]
> Yes, the open targets shouldn’t be opened before the end of the battle.

ok, but that doesn't address copy/paste asap nonsense, does it? just delaying that in time

**Comment by Esger** [comment]
Copy/paste is OK for open targets AFTER the battle ends. I suggested this before: all the unique top 50 or 100 solutions should be visible to learn from.
